.github/workflows/ci.yml

```diff
@@ -34,7 +32,7 @@ jobs:
           - tlsn
           - notary-server
         include:
-          - package: components/integration-tests
+          - package: components/tls/tls-mpc
```

Comment:
This will run the tests in `tls-mpc` twice, since they are already in `tls` above
See test run at https://github.com/tlsnotary/tlsn/actions/runs/9658293628/job/26639185063?pr=517